Dashboard/src/components/Github.js

```diff
@@ -21,16 +21,18 @@ class Github extends Container {
               const commitDate = moment(entry.author.date).format("MMM DD@HH:mm");
               return (
                 <div key={entry + i} className="github-entry">
-                  <img className="github-icon" src={github} alt="Github icon" />
-                  <span className="github-name" >{entry.author.name}</span>
-                  <span> committed to </span>
-                  <a 
-                    href={`https://github.com/Seneca-CDOT/${entry.repoName}/tree/${entry.branchName}`}  
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                  <span className="github-repo github-link"> {` ${entry.repoName}/${entry.branchName}:`}</span>
-                  </a>
+                  <div className="github-entry-details">
+                    <img className="github-icon" src={github} alt="Github icon" />
+                    <span className="github-name" >{entry.author.name}</span>
+                    <span> committed to </span>
+                    <a 
+                      href={`https://github.com/Seneca-CDOT/${entry.repoName}/tree/${entry.branchName}`}  
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                    <span className="github-repo github-link"> {`${entry.repoName}/${entry.branchName}:`}</span>
```

Comment:
Is there a reason we have 2 css classes `github-repo github-link` here?

Reply:
There appears to be duplicate classes of `github-repo` in `App.css`. `Osteppy` component seems to use some of the github classes as well. These should both be addressed.